build.gradle

```diff
@@ -150,6 +152,62 @@ task wrapper(type: Wrapper) {
     gradleVersion = '2.3'
 }
 
+task cleanDb << {
+    def sql = getSqlInstance()
+
+    sql.execute('TRUNCATE zsr_data.event_type')
+
+    sql.close()
+}
+
+task dbBootstrap << {
+    def sql = getSqlInstance()
+
+    def sqlFileList = getSortedSqlFiles()
+
+    sqlFileList.each {
+        println 'Running script: ' + it
+        String sqlString = it.text
+        sql.execute(sqlString)
+    }
+
+    sql.execute("RESET ROLE")
+
+    sql.execute("CREATE ROLE schemaregistry_app with login password 'schemaregistry'")
+
+    sql.execute("GRANT zalando_schemaregistry_data_writer TO schemaregistry_app")
+
+    sql.close()
+}
+
+def getSortedSqlFiles() {
+    def sqlFileList = []
+    new File('database').eachFileRecurse(groovy.io.FileType.FILES) {
+        if (it.name.endsWith('.sql')) {
+            sqlFileList << it
+        }
+    }
+
+    sqlFileList.sort()
+}
+
+def getSqlInstance() {
+    // This is needed to get postgreSQL driver onto the Groovy/Gradle classpath
+    configurations.pgsql.each { file ->
+        println "Adding URL: $file"
+        gradle.class.classLoader.addURL(file.toURI().toURL())
+    }
+
+    def db = [
```

Comment:
just wondering if we can somehow grab those settings from https://github.com/zalando/nakadi/blob/ARUHA-50-db-bootstrap/src/main/resources/application.yml#L8 to have that in one place only